spring-data-multi-storage-transaction-sample/README.md

```diff
@@ -8,77 +8,68 @@ This tutorial describes how to create a sample Spring Boot application by using
 - Gradle
 - Docker, Docker Compose
 
+You also need access for [ScalarDB SQL GitHub repository](https://github.com/scalar-labs/scalardb-sql) and [Packages in ScalarDB SQL repository](https://github.com/orgs/scalar-labs/packages?repo_name=scalardb-sql).
+Since they are available under a commercial license, you need to get the license and permission to access them.
+For more details, please [contact us](https://scalar-labs.com/contact_us/).
+
+You also need the `gpr.user` property for your GitHub username and the `gpr.key` property for your personal access token.
+So you need to have the properties in `~/.gradle/gradle.properties`, or specify the properties with the `-P` option when running the `./gradlew` command as follows:
+
+```shell
+$ ./gradlew run ... -Pgpr.user=<your GitHub username> -Pgpr.key=<your personal access token>
+````
+
+Please read [Install - ScalarDB SQL](https://github.com/scalar-labs/scalardb-sql#install) for more detail.
+
 ## Sample application
 
 ### Overview
 
 This tutorial describes how to create a sample Spring Boot application for the same use case as [ScalarDB Sample](https://github.com/scalar-labs/scalardb-samples/tree/main/scalardb-sample) but by using Spring Data JDBC for ScalarDB with Multi-storage Transaction.
 Please note that application-specific error handling, authentication processing, etc. are omitted in the sample application since this tutorial focuses on explaining how to use Spring Data JDBC for ScalarDB with Multi-storage Transaction.
-For details, please see [Guide of Spring Data JDBC for ScalarDB](https://scalar-labs.github.io/scalardb-sql/spring-data-guide.html).
+For details, please see [Guide of Spring Data JDBC for ScalarDB](https://github.com/scalar-labs/scalardb-sql/blob/main/docs/spring-data-guide.md).
```

Comment:
The document isn't public, right? On the other hand, I think this README is public. So the official document should be used here?

Reply:
Currently, only users with access to the ScalarDB SQL GitHub repository can execute the sample code. Thus, we can direct them to the repository. Additionally, the GitHub IO pages will be removed, as the Doc site is now open.

Reply:
That makes sense. I forgot the future removal of the GitHub IO pages.

On the other hand, I think showing prospective users the sample code and related documents including `Guide of Spring Data JDBC for ScalarDB` motivates them to use the products. How about https://scalardb.scalar-labs.com/docs/3.9/scalardb-sql/spring-data-guide/ ?

Reply:
Yeah, we can use the Doc site page.

In the near future, we will put the Docker image, the Java client library, and the SQL CLI tool to a location accessible to users. At that point, I think we can change the GitHub link to point to the documentation site instead. Is that okay?

Reply:
SGTM!